sdk/core/azure-core/src/http/retry_policy.cpp

```diff
@@ -93,16 +93,17 @@ Context::Key const RetryKey;
 
 int32_t RetryPolicy::GetRetryCount(Context const& context)
 {
-  if (!context.HasKey(RetryKey))
+  int number = -1;
+  if (!context.TryGetValue<int>(RetryKey, number))
```

Comment:
```suggestion
  if (!context.TryGetValue<int32_t>(RetryKey, number))
```

Reply:
I am fine with `int`, however RetryOptions has it as `int32_t`, they should be in sync.